### PR TITLE
IsZeroPack Hint Implementation

### PR DESCRIPTION
**Pull request by Sh0g0-1758**
Fixes: #464

### CODE REVIEW

File: pkg/hintrunner/zero/zerohint_signature.go [hunk 1]
```diff
@@ -386,3 +386,55 @@ func createDivModNPackedDivmodV1Hinter(resolver hintReferenceResolver) (hinter.H
...
+			if err != nil {
+				return err
+			}
+
+			xValues, err := vm.Memory.ResolveAsBigInt3(xAddr)
+			if err != nil {
+				return err
+			}
+
+			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack
```
Comment (TAdev0) [inline pkg/hintrunner/zero/zerohint_signature.go:413]: if you want to add the comments line by line when implementing, can you do it for all lines of the hint?  i.e., add `//> x = pack(ids.x, PRIME) % SECP_P`

**Review by TAdev0 — CHANGES_REQUESTED** [review]

File: pkg/hintrunner/zero/zerohint_signature.go [hunk 2]
```diff
@@ -386,3 +386,55 @@ func createDivModNPackedDivmodV1Hinter(resolver hintReferenceResolver) (hinter.H
 
 	return newDivModNPackedDivmodV1Hint(a, b), nil
 }
+
+// IsZeroPack hint caculates packed value modulo SECP_P prime
```
[the remark this answers is not in the document]
Comment (Sh0g0-1758) [reply pkg/hintrunner/zero/zerohint_signature.go:390]: done.

File: pkg/hintrunner/zero/zerohint_signature.go (hunk 1 above)
Comment (Sh0g0-1758) [reply pkg/hintrunner/zero/zerohint_signature.go:413]: Hm.. It was just a design choice I borrowed from another hint impl. I guess I will just remove it. The current impl is intuitive enough.

**Review by har777 — APPROVED** [review]

File: pkg/hintrunner/zero/zerohint_signature.go [hunk 3]
```diff
@@ -386,3 +386,54 @@ func createDivModNPackedDivmodV1Hinter(resolver hintReferenceResolver) (hinter.H
...
+			if err != nil {
+				return err
+			}
+
+			xValues, err := vm.Memory.ResolveAsBigInt3(xAddr)
+			if err != nil {
+				return err
+			}
+
+			secpBig, _ := secp_utils.GetSecPBig()
```
[the remark this answers is not in the document]
Comment (Sh0g0-1758) [reply pkg/hintrunner/zero/zerohint_signature.go]: done.

**Pull request merged**